### DISCUSSION TITLE
Experiments on AISHELL-4

### DISCUSSION THREAD

**Discussion by hbredin** [pyannote/pyannote-audio#858] [General]
@desh2608 [asked](https://github.com/felixfuyihui/AISHELL-4/issues/7) about diarization results on AISHELL-4. Thought I'd open a dedicated discussion on pyannote repo.

> @hbredin Did you got around to running some diarization baselines on this data?

Still on my TODO list. [Wanna try](https://github.com/pyannote/pyannote-audio/tree/develop#tldr)? 

> I ran VBx and spectral clustering and got the following results:
> Method 	MS 	FA 	Conf. 	DER
> VBx 	17.06 	1 	3.97 	22.03
> Spectral 	17.06 	1 	3.11 	21.17
> 
> I was wondering if you have any overlap detectors trained on Mandarin?

Did you try [pyannote/segmentation](https://huggingface.co/pyannote/segmentation) with the [overlapped speech detection](https://huggingface.co/pyannote/overlapped-speech-detection) pipeline?   
Should work slightly better than goold old `ovl-dihard` and `ovl-ami`.

[Fine-tuning](https://github.com/pyannote/pyannote-audio/blob/develop/tutorials/training_a_model.ipynb) `pyannote/segmentation` on AISHELL-4 would definitely help, though.

**Comment by desh2608** [comment]
Thanks for the suggestions! I'll try the segmentation pipeline and see if it helps. Will get back with updates soon hopefully.
